### PR TITLE
ci: include e2e tests into our GitHub workflow (GitHub actions)

### PR DESCRIPTION
**Pull request by Ebmili**
## Jira issue

JIRA issue: [FSU-XX](https://klau.atlassian.net/browse/FSU-XX)

## Checklist before requesting a review

- [ ] Checked which branch you would merge.
- [ ] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [ ] Performed a self-review of code (bugs, clean code rules, etc).
- [ ] Checked if added variables have unique names.
- [ ] Checked if added code/function is not a duplicate. 
- [ ] Tests have been added if possible.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

**Comment by im-klau** [comment]
@Ebmili as you can see this check:
Zrzut ekranu 2022-12-7 o 14 50 59
failed because of the Pull Request name that doesn't follow the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) rules.

or the Angular one:
Zrzut ekranu 2022-12-7 o 14 52 45

could you take a look at the assets (image, URL) I attached here and fix the name of this PR? 🙏 

It would be nice to apply the rules to commits as well :D

**Comment by Ebmili** [comment]
> @Ebmili as you can see this check: Zrzut ekranu 2022-12-7 o 14 50 59 failed because of the Pull Request name that doesn't follow the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) rules.
> 
> or the Angular one: Zrzut ekranu 2022-12-7 o 14 52 45
> 
> could you take a look at the assets (image, URL) I attached here and fix the name of this PR? 🙏
> 
> It would be nice to apply the rules to commits as well 

 I've corrected PR name, but not sure If I should correct commit message as well?

**Comment by im-klau** [comment]
> I've corrected PR name, but not sure If I should correct commit message as well?

@Ebmili while doing new commits just remember about all of the `fix, feat, ci` etc based on the commit naming convention. There's no need to change the old commit messages now :D

File: .github/workflows/ci-lint-and-jest.yml [hunk 1]
```diff
@@ -28,3 +28,8 @@ jobs:
 
       - name: Tests
         run: yarn test
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v3
+        with:
+        command: yarn cypress:run
```
Comment (im-klau) [inline .github/workflows/ci-lint-and-jest.yml:35]: It looks like this line should have additional indentation. 
Actually, in the `yml` it's always about indentations, if there's a wrong one -> the code won't work 😅 

```yml
- name: Cypress run
  uses: cypress-io/github-action@v3
  with:
    command: yarn cypress:run
```

**Comment by im-klau** [comment]
@Ebmili just one, small change is needed here 🙏

File: .github/workflows/ci-lint-and-jest.yml (hunk 1 above)
Comment (Ebmili) [reply .github/workflows/ci-lint-and-jest.yml]: @im-klau this is done :)

**Review by im-klau — APPROVED** [review]

**Pull request merged**